### PR TITLE
Renaming #14

### PR DESCRIPTION
**Pull request by avetall87**

### CODE REVIEW

File: src/main/java/ru/spb/devclub/utils/db/ProxyResultSet.java [hunk 1]
```diff
@@ -8,7 +8,8 @@
 import java.util.Calendar;
 import java.util.Map;
 
-/* package */abstract class ProxyResultSet implements ResultSet {
+/* package */
```
Comment (alexengrig) [inline src/main/java/ru/spb/devclub/utils/db/ProxyResultSet.java]: Inline or remove?

Comment (alexengrig) [inline src/main/java/ru/spb/devclub/utils/db/ProxyResultSet.java]:
```suggestion
```

**Comment by avetall87** [comment]
Fixes #14

**Comment by avetall87** [comment]
Fixes #14

**Comment by avetall87** [comment]
close #15

**Comment by avetall87** [comment]
clo

**Pull request merged**